spec/support/unit/matchers/fail_with_message_matcher.rb

```diff
@@ -36,10 +38,9 @@ def failure_message_for_should
       end
 
       def failure_message_when_negated
-        msg  = "Expectation should not have failed with message '#{expected}'"
-        msg << ", but did."
-
-        msg
+        lines = ["Expectation should not have failed with message:"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.